lib/ditty/utils.py

```diff
@@ -0,0 +1,9 @@
+def convertSecondsToStringTime(seconds): 
```

Comment:
Would you mind converting this name to "convert_seconds_to_string_time" just to match the general library style? Longer term I'm going to add a style precommit hook so that things will auto format.

Reply:
I was programming a Flutter application while doing this code (Camel case required), I still can't believe I did it hahahaha.